pkg/ccl/backupccl/utils_test.go

```diff
@@ -113,6 +114,12 @@ func backupRestoreTestSetupWithParams(
 	}
 
 	cleanupFn := func() {
+		if tc.Conns[0].Ping() == nil {
+			if err := backuputils.CheckForInvalidDescriptors(tc.Conns[0]); err != nil {
+				t.Fatal(err)
+			}
+			t.Log("No Invalid Descriptors")
```

Comment:
nit: no invalid descriptors

also curious, what's this Ping about?

Reply:
See the updated CheckForInvalidDescriptors method.